src/pal/tools/gen-buildsys-clang.sh

```diff
@@ -40,7 +42,11 @@ else
   exit 1
 fi
 
-desired_llvm_version=3.5
+if [ $OS = "FreeBSD" ]; then
+	desired_llvm_version=35
+else
+	desired_llvm_version=3.5
+fi
```

Comment:
It might be better to break this into __LLVM_major and _LLMV_minor, and then do platform specific concatenation.  The current mess is getting a bit soupy.